app/domain/search/audit_filter.rb

```diff
@@ -0,0 +1,42 @@
+class Search
+  class AuditFilter
+    AUDIT_STATUSES_IDENTIFIERS = [
+      :audited,
+      :non_audited,
+    ].freeze
```

Comment:
I was thinking this could be a binary thing, e.g. `audited=true` / `audited=false` but thinking about it, this is probably better if we have more states like 'in progress'. 👍